src/Components/Comments.jsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from "react";
+import { getCommentsByArticleId } from "../utils/api";
+import { useParams } from "react-router";
+import { ListGroup } from "react-bootstrap";
+
+export default function Comments() {
+  const { article_id } = useParams();
+  const [comments, setComments] = useState({});
+  const [isLoading, setIsLoading] = useState(true);
+  useEffect(() => {
+    getCommentsByArticleId(article_id).then((comment) => {
```

Comment:
It isn't a single comment. Your naming convention should reflect what the data actually is. If you want to ensure it doesn't conflict with the state called `comments`, you could call this one `commentsFromApi`